locations/parzelle17.yml

```diff
@@ -0,0 +1,78 @@
+---
+location: parzelle17
+location_nice: Parzelle 17
+latitude: 52.478675
+longitude: 13.471268
+community: true
+
+dns_servers:
+  # quad9
+  - 9.9.9.9
+  - 149.112.112.112
+  - 2620:fe::fe
+  - 2620:fe::9
+  # cloudflare
+  - 1.1.1.1
+  - 1.0.0.1
+  - 2606:4700:4700::1111
+  - 2606:4700:4700::1001
+
+hosts:
+  - hostname: parzelle17-core
+    role: corerouter
+    model: "dlink_dap-x1860-a1"
+    wireless_profile: freifunk_default
+
+ipv6_prefix: '2001:bf7:840:f00::/56'
+
+# got following prefixes:
+# Router: ?
+# --MGMT: 10.31.207.1/28
+# --MESH: ?
+# --DHCP: 10.31.207.33/27
```

Comment:
this needs to be `10.31.207.0/28` and `10.31.207.32/27` - same for the vlan definitions below. The network "name" is the zero address, and the first usable address is address one.